scripts/tatt

```diff
@@ -93,6 +92,28 @@ parser.add_option("-v", "--verbose",
 config['verbose']=options.verbose
 # Normalize the template dir:
 config['template-dir']=os.path.abspath(config['template-dir'])+os.sep
+
+# Now that we use nattka, we need a working repodir
+if config['repodir'] == "":
+    if os.path.isdir("/var/db/repos/gentoo"):
+        config['repodir']="/var/db/repos/gentoo"
```

Comment:
spaces around =

Reply:
I adopted existing coding style, see assignment in line 92 and 94 above.